### DISCUSSION TITLE
Add a failsafe for when rendering the _error page fails

### DISCUSSION THREAD

**Discussion by sin-ack** [vercel/next.js#30526] [Ideas]
### Describe the feature you'd like to request

There are cases where rendering the error page itself can fail. In these cases, Next.js currently tries to render the error page over and over again in an infinite loop. Here, a good addition would be to have a bare-bones failsafe error page which doesn't depend on any part of the application itself, and gives a concise explanation.

### Describe the solution you'd like

When rendering the `_error` page fails, Next.js should render a simple component that doesn't depend on any part of the application, with a possibly user-defined error message. It could be similar to the current `Error` component, with the possibility of replacing the string being displayed via `next.config.js`.

### Describe alternatives you've considered

An alternative could be to catch cases where `_error` fails to render in development and provide a descriptive error message. In my case the error was only caught in production because the error in development was masked by the other errors that I was already attempting to solve. Something like a suggestion to add an error boundary component could be printed in the console or as an error message in the error popup window.

**Comment by timneutkens** [comment]
> Next.js currently tries to render the error page over and over again in an infinite loop

Are you using the latest version? It should fall back to the default error page automatically. Can you provide a reproduction?
